backend/peermatching.js

```diff
@@ -0,0 +1,25 @@
+function matchpeer(questions, students, numTask) {
```

Comment:
Consider moving this file to a folder called peer-reviews/ or util/, depending on what you like